miso-web/src/main/webapp/scripts/list_servicerecord.js

```diff
@@ -114,7 +115,17 @@ ListTarget.servicerecord = {
             list += "</ul>";
             return list;
           }
-          return data;
+          var filenames = "";
```

Comment:
This could also be represented as 
```
full.attachments.map(f => f.filename).join(', ');
```
(@djcooke I think comma-separated might be slightly clearer in cases where the file names have spaces, but space-separated like you had recommended would be easily understood)

Reply:
This string is for filtering purposes, and is not displayed to the user. Commas should not be added, as they would alter what gets matched

Reply:
Fair enough! Space-separated it is.